tests/eas/load_tracking.py

```diff
@@ -537,6 +540,206 @@ def test_load_avg_behaviour(self, experiment, tasks):
         """
         return self._test_behaviour(experiment, tasks, 'load_avg')
 
+class _CPUMigrationBase(LisaTest):
+    """Base class for shared functionality of tasks migration tests"""
+    test_conf = {
+        'tools'    : [ 'rt-app' ],
+        'ftrace' : {
+            'events' : [
+                'sched_switch',
+                'sched_load_cfs_rq',
+            ],
+        },
+        'modules': ['cpufreq', 'cgroups'],
+    }
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(_CPUMigrationBase, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def generate_json(cls, test_env, tasks, json_file_name):
+        """
+        Generate a json file which describes the different tasks.
+        :param test_env: Environment of the test
+        :type test_env: TestEnv
+
+        :param tasks: lists of task
+        :type tasks:[RTATask]
+
+        :param json_file_name: name of the json file
+        :type json_file_name: str
+        :return the generate file name without extensin
+        """
+        rtapp = RTA(test_env.target, json_file_name,
+            calibration=test_env.calibration())
+        return rtapp.conf(kind='profile',
+            params={'task'+str(i): task.get() for i, task in enumerate(tasks)},
+            run_dir=test_env.target.working_directory+'/run_dir/')
```

Comment:
Done: use directly test_env.target.working_directory

Reply:
This was not working because the test executor uses another path for the log directory (os.path.join(test_env.target.working_directory, '/run_dir/'). So to manage to get the value given by the executor a static function has been added to the executor to get this run_dir. It can be now used here to get the good run_dir